swagger_server/itm/data/metrics/test/sample.yaml

```diff
@@ -108,7 +108,7 @@ state:
         mental_status: UNRESPONSIVE # controlled vocab includes AGONY, CALM, CONFUSED, SHOCK, UPSET, UNRESPONSIVE
         breathing: FAST # controlled vocab includes NORMAL, FAST, RESTRICTED, NONE
         heart_rate: FAST # controlled vocab includes NONE, FAINT, NORMAL, FAST
-        spo2: 92 # a percentage
+        spo2: "NORMAL"
```

Comment:
The sample YAML also includes the controlled vocab, so please add:
`# controlled vocab includes NONE, LOW, NORMAL`
to all of the `spo2` properties in (only) this file.